apps/angular/projection/src/app/component/city-card/city-card.component.ts

```diff
@@ -1,13 +1,55 @@
 import { Component, OnInit } from '@angular/core';
+import {
+  FakeHttpService,
+  randomCity,
+} from '../../data-access/fake-http.service';
+import { CityStore } from '../../data-access/city.store';
+import { City } from '../../model/city.model';
+import { CardComponent } from '../../ui/card/card.component';
+import { ListItemComponent } from '../../ui/list-item/list-item.component';
+import { CardItemContentDirective } from '../../ui/card/card-item-content.directive';
 
 @Component({
   selector: 'app-city-card',
-  template: 'TODO City',
+  template: ` <app-card
+    [list]="cities"
+    class="bg-light-blue"
+    (addItem)="addNewCity()">
+    <ng-template let-city appCardItemContent>
+      <app-list-item (deleteItem)="deleteCity(city.id)">
+        {{ city.name }}
+      </app-list-item>
+    </ng-template>
+  </app-card>`,
+  styles: [
+    `
+      .bg-light-blue {
+        background-color: rgba(0, 0, 250, 0.1);
+      }
+    `,
+  ],
   standalone: true,
-  imports: [],
+  imports: [CardComponent, ListItemComponent, CardItemContentDirective],
 })
 export class CityCardComponent implements OnInit {
-  constructor() {}
+  cities: City[] = [];
 
-  ngOnInit(): void {}
+  constructor(
+    private http: FakeHttpService,
+    private store: CityStore,
+  ) {}
+
+  ngOnInit(): void {
+    this.http.fetchCities$.subscribe((s) => this.store.addAll(s));
+
+    this.store.cities$.subscribe((s) => (this.cities = s));
```

Comment:
prefer declarative over imperative. 

You can even use signals

Reply:
@tomalaforge fixed